src/ast_generic/Meta_AST.ml

```diff
@@ -803,6 +803,7 @@ and vof_attribute = function
   | OtherAttribute (v1, v2) ->
       let v1 = vof_todo_kind v1 and v2 = OCaml.vof_list vof_any v2 in
       OCaml.VSum ("OtherAttribute", [ v1; v2 ])
+  | DocStringAttr (st, _tok) -> OCaml.VSum("DocStringAttr", [VString st])
```

Comment:
Thanks for not using `v1` and `v2` as variable names. 😅 [That said, `st` sounds like `statement`, maybe `str` for string ? or `doc` ?]